### PR TITLE
Feature input component

### PR DESCRIPTION
**Pull request by alfonsom-cogent**

### CODE REVIEW

File: components/button.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react'
+
+interface Props {
+  text: string
+  type: string
+  size: string
+}
+
+export const Button: FC<Props> = ({ text, type, size }) => {
+  let styleType: string = type + size
```
Comment (n1c01a5) [inline components/button.tsx:10]: better to use template string

File: components/input.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  labelText,
+  placeholder,
+  type,
+  required,
+  disabled,
+  value,
+  onChange,
+}) => {
+  let inputStyle: string =
+    'form-control block w-full px-4 py-2 text-base font-normal text-gray-600 bg-white bg-clip-padding border border-solid border-[#FFC905] rounded-full hover:shadow-lg transition ease-in-out m-0 focus:text-black focus:bg-white focus:border-[#FF8C05] focus:outline-none'
```
Comment (n1c01a5) [inline components/input.tsx:23]: use double quotes

**Review by n1c01a5 — APPROVED** [review]

File: components/button.tsx (hunk 1 above)
Comment (alfonsom-cogent) [reply components/button.tsx:10]: If we use a template string instead, the deployment fails

**Review by n1c01a5 — APPROVED** [review]

File: components/button.tsx (hunk 1 above)
Comment (n1c01a5) [reply components/button.tsx]: mmm that's weird

**Pull request merged**